docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -2,10 +2,9 @@
 
 ## Platform requirements
 
-- A system with support for SGX1 with Flexible Launch Control (FLC).
-Note: To check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md).
+- An Intel system.
```

Comment:
This could be any x64 system (both Intel/AMD).

The simulation mode does require certain instructions like xsave etc. But quite hard to tell which versions of the Intel/AMD processors do support them.